docs/source/_static/embed-bundle.js

```diff
@@ -0,0 +1,810 @@
+define("jupyter-matplotlib", ["@jupyter-widgets/base"], (__WEBPACK_EXTERNAL_MODULE__jupyter_widgets_base__) => { return /******/ (() => { // webpackBootstrap
```

Comment:
woops this file snuck in at some points. Will need to remove this in a rebase at some point prior to merging